### PR TITLE
Updates for running GMRES example with half precision

### LINKED ISSUE #1049: Test GMRES example with half_t
Depends on https://github.com/kokkos/kokkos-kernels/pull/1028.

### PR DESCRIPTION
**Pull request by e10harvey**
Fixes #1049.

### CODE REVIEW

File: example/gmres/test_real_A.cpp [hunk 1]
```diff
@@ -55,19 +55,23 @@
 
 int main(int /*argc*/, char ** /*argv[]*/) {
 
-  typedef double                             ST;
+  using ST = Kokkos::Experimental::half_t;
+  //using ST = double;
```
Comment (jennloe) [inline example/gmres/test_real_A.cpp:59]: Let's remove the half_t line?  Or make the half_t into a separate test?  I really want to leave this test in double precision.

File: example/gmres/test_real_A.cpp [hunk 2]
```diff
@@ -55,19 +55,23 @@
...
   typedef int                               OT;
-  typedef Kokkos::DefaultExecutionSpace     EXSP;
+  using EXSP = Kokkos::Serial;
```
Comment (jennloe) [inline example/gmres/test_real_A.cpp:61]: Change back to default for this test.

File: example/gmres/ex_real_A.cpp [hunk 3]
```diff
@@ -54,7 +54,7 @@
 
 int main(int argc, char *argv[]) {
 
-  typedef double                            ST;
+  typedef Kokkos::Experimental::half_t      ST;
```
Comment (jennloe) [inline example/gmres/ex_real_A.cpp:57]: Let's change this back to double?  I don't want to push this example with half precision.

File: example/gmres/gmres.hpp [hunk 4]
```diff
@@ -50,8 +50,22 @@
 #include<KokkosBlas3_trsm.hpp>
 #include<KokkosSparse_spmv.hpp>
 
+
+////////////////////////////////////////////////////////////////////////////////
+// libstdc++ half_t overloads
+////////////////////////////////////////////////////////////////////////////////
+#if !KOKKOS_HALF_T_IS_FLOAT
```
Comment (jennloe) [inline example/gmres/gmres.hpp:57]: I'm okay with putting this here, but would it make more sense in a separate file?

File: example/gmres/test_real_A.cpp [hunk 5]
```diff
@@ -55,19 +55,23 @@
...
+  using sp_matrix_type = KokkosSparse::CrsMatrix<ST, OT, EXSP>;
   using ViewVectorType = Kokkos::View<ST*,Kokkos::LayoutLeft, EXSP>;
   typedef sp_matrix_type::non_const_ordinal_type ncOT;
   typedef sp_matrix_type::const_ordinal_type cOT;
   typedef sp_matrix_type::non_const_size_type ncST;
 
   std::string ortho("CGS2"); //orthog type
   int m = 15; //Max subspace size before restarting.
-  double convTol = 1e-10; //Relative residual convergence tolerance.
+  ST convTol = 1e-1; //Relative residual convergence tolerance.
```
Comment (jennloe) [inline example/gmres/test_real_A.cpp:74]: Change back to 1e-10.

**Review by jennloe — CHANGES_REQUESTED** [review]

**Comment by jennloe** [comment]
@e10harvey I think you'll have to pull in my most recent changes and then add these fixes on top.  (I changed the GMRES parameter list in a previous PR.  Shouldn't be too bad of a fix.)

Also, did the corresponding fixes for half_t get merged into Kokkos?

**Comment by e10harvey** [comment]
> Also, did the corresponding fixes for half_t get merged into Kokkos?

Not yet. The reviewers are still looking over the changes.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - GitHub reports Mergeable status = False

File: example/gmres/gmres.hpp (hunk 4 above)
Comment (e10harvey) [reply example/gmres/gmres.hpp:57]: this should reside in kokkos core.

Comment (e10harvey) [reply example/gmres/gmres.hpp:57]: i will leave this here for now but remove it once we add it to kokkos core.

**Comment by e10harvey** [comment]
@jennloe: Thank you for your feedback and help with testing this. All your feedback and the conflicts should be addressed. Sorry about the formatting changes; KokkosKernels will be enforcing code style via CI soon and my IDE is already applying this formatting.

**Review by jennloe — APPROVED** [review]
Looks good now!

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - User Requested Retest - Label AT: RETEST will be reset after testing.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light</h3>
<ul>
  <li>Build Num: 13</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720</h3>
<ul>
  <li>Build Num: 366</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 13</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720</h3>
<ul>
  <li>Build Num: 357</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA10</h3>
<ul>
  <li>Build Num: 14</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL18</h3>
<ul>
  <li>Build Num: 343</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA10_LayoutRight</h3>
<ul>
  <li>Build Num: 14</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA9</h3>
<ul>
  <li>Build Num: 13</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720_GCC740</h3>
<ul>
  <li>Build Num: 14</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: KOKKOSKERNELS (e10harvey/kokkos-kernels)</dt>
  <ul>
    <li>Branch: issue1049</li>
    <li>SHA: 6cf9a2275b67a32f9b6202c80310a8bef94472fd</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: e10harvey
</details>

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light</h3>
<ul>
  <li>Build Num: 13</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720</h3>
<ul>
  <li>Build Num: 366</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light_LayoutRight</h3>
<ul>
  <li>Build Num: 13</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720</h3>
<ul>
  <li>Build Num: 357</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA10</h3>
<ul>
  <li>Build Num: 14</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL18</h3>
<ul>
  <li>Build Num: 343</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA10_LayoutRight</h3>
<ul>
  <li>Build Num: 14</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA9</h3>
<ul>
  <li>Build Num: 13</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720_GCC740</h3>
<ul>
  <li>Build Num: 14</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>issue1049</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/e10harvey/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>6cf9a2275b67a32f9b6202c80310a8bef94472fd</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>541b56502f46717f38cad1bc96aae88f44b3dfcd</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: RETEST</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1067</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>
</details>

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Merge Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ jennloe ]!

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Pull Request MUST BE MERGED MANUALLY BY Project Team - This Repo does not support Automerge

**Pull request merged**